_source/_includes/metric-shipping/telegraf-setup-linux.md

```diff
@@ -0,0 +1,73 @@
+#### Ubuntu & Debian
+
+##### Add the InfluxData repository 
+
+###### Ubuntu
+
+Add the InfluxData repository with the following commands:
+
+```shell
+wget -qO- https://repos.influxdata.com/influxdb.key | sudo apt-key add -
+source /etc/lsb-release
+echo "deb https://repos.influxdata.com/${DISTRIB_ID,,} ${DISTRIB_CODENAME} stable" | sudo tee /etc/apt/sources.list.d/influxdb.list
+```
+
+###### Debian
+
+Add the InfluxData repository with the following commands:
```

Comment:
Do we really need those parts ( the influx repository and the influx key) ?
If customer want to use telegraf cant he just  run
sudo apt-get update && sudo apt-get install telegraf